packages/components/carousel/__tests__/carousel.spec.tsx

```diff
@@ -2,27 +2,49 @@ import { nextTick, reactive } from 'vue'
 import { mount } from '@vue/test-utils'
 import Carousel from '../src/carousel.vue'
 import CarouselItem from '../src/carousel-item.vue'
+
+import type { VueWrapper } from '@vue/test-utils'
 import type { CarouselInstance } from '../src/carousel'
 
 const wait = (ms = 100) =>
   new Promise((resolve) => setTimeout(() => resolve(0), ms))
 
-const generateCarouselItems = (count = 3) => {
+const generateCarouselItems = (count = 3, hasLabel = false) => {
   const list = Array.from({ length: count }, (_, index) => index + 1)
-  return list.map((i) => <CarouselItem key={i}></CarouselItem>)
+  return list.map((i) =>
+    hasLabel ? <CarouselItem key={i} label={i} /> : <CarouselItem key={i} />
+  )
 }
 
 describe('Carousel', () => {
-  it('create', () => {
-    const wrapper = mount({
+  let wrapper: VueWrapper<any>
+
+  const createComponent = (
+    props: any = {},
+    count?: number,
+    hasLabel?: boolean
+  ) => {
+    return mount({
       setup() {
         return () => (
           <div>
-            <Carousel ref="carousel">{generateCarouselItems()}</Carousel>
+            <Carousel {...props}>
```

Comment:
Remove `ref=carousel`?

Reply:
I extracted the props as parameter, whenever test case needs it, it can pass that as parameter instead of add it by default.